ci-operator/step-registry/openshift/image-based/install/openshift-image-based-install-commands.sh

```
@@ -86,12 +124,13 @@ EOF
 
 chmod +x ${SHARED_DIR}/image_based_install.sh
 
-echo "Transfering install script..."
-scp "${SSHOPTS[@]}" ${SHARED_DIR}/image_based_install.sh $ssh_host_ip:$remote_workdir
+echo "Transferring install script..."
+ssh_retry scp "${SSHOPTS[@]}" ${SHARED_DIR}/image_based_install.sh $ssh_host_ip:$remote_workdir
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)